src/app/wallet/v2/views/asset_list.nim

```diff
@@ -22,6 +30,9 @@ QtObject:
   proc newAssetList*(): AssetList =
     new(result, delete)
     result.assets = @[]
+    result.propertiesList = newTraitsList()
+    result.rankingList = newTraitsList()
+    result.statsList = newTraitsList()
```

Comment:
Not sure here, but should you not also delete the 3 nested lists?

Reply:
definitely . My bad